terminus-ui/table/src/table.component.scss

```diff
@@ -1,6 +1,7 @@
 @import './../../scss/helpers/color';
```

Comment:
since scrollbars includes color, this can be removed.

Reply:
I don't want us to depend on magic imports like that. I would prefer we remain explicit to avoid unexpected issues during later changes.